pkg/app/crd_app.go

```diff
@@ -119,10 +119,100 @@ func (a *CRDApp) updateApp(updateFunc func(*kcv1alpha1.App)) error {
 	return nil
 }
 
-func (a *CRDApp) Reconcile() (reconcile.Result, error) {
-	return a.app.Reconcile()
+func (a *CRDApp) Reconcile(force bool) (reconcile.Result, error) {
+	return a.app.Reconcile(force)
 }
 
 func (a *CRDApp) watchChanges(callback func(kcv1alpha1.App), cancelCh chan struct{}) error {
 	return NewCRDAppWatcher(*a.appModel, a.appClient).Watch(callback, cancelCh)
 }
+
+// Get all SecretRefs from App spec
+func (a *CRDApp) GetSecretRefs() map[string]struct{} {
```

Comment:
we can probably push this down to App CR, but still expose it on CRDApp. tests could go against App CR's methods.

Reply:
Moved to App CR